hawc/apps/common/table.py

```diff
@@ -0,0 +1,60 @@
+from html.parser import HTMLParser
+from typing import List
+
+from docx import Document
+from pydantic import BaseModel, conint, validator
+
+
+class GenericTableCell(BaseModel):
+    row: conint(ge=0)
+    column: conint(ge=0)
+    html_text: str
```

Comment:
may as well add `col_span: int=0` and `row_span: int = 0` as thats something people will always want to do

Reply:
or maybe it's an optional `conint(ge=2)`?

Reply:
might also need validation check to ensure column + col_span < tbl.columns or something similar to that